### PR TITLE
[SE-219] Added examples using Helm

### PR DESCRIPTION
**Pull request by bodo-sai**
Instead of manually installing CRDs and yaml files, the users can use helm to run bodo jobs in kubernetes.

### CODE REVIEW

File: Kubernetes/README.md [hunk 1]
```diff
@@ -16,12 +16,60 @@ A docker image created from this Dockerfile is also available on DockerHub: [bod
...
+git clone https://github.com/Bodo-inc/Bodo-examples.git
+```
+
+#### Step 2: Setup Helm and run 
+
+Check if Helm is installed in your system using `helm version`. If not, check this [guide](https://helm.sh/docs/intro/install/) for setting up Helm in your system.
+
+To run the sample bodo example, Go to the clone repository and run the following command
+```
+helm install <release-name> Kubernetes/helm
```
Comment (sahil1105) [inline Kubernetes/README.md:49]: What's release-name?

File: Kubernetes/helm/crds/mpi-operator.yaml [hunk 2]
```diff
@@ -0,0 +1,485 @@
...
+  group: kubeflow.org
+  names:
+    kind: MPIJob
+    plural: mpijobs
+    shortNames:
+    - mj
+    - mpij
+    singular: mpijob
+  scope: Namespaced
+  versions:
```
Comment (sahil1105) [inline Kubernetes/helm/crds/mpi-operator.yaml:39]: What are these versions exactly?

**Review by sahil1105 — COMMENTED** [review]
Looks good overall. Thanks @bodo-sai.

Have we asked someone to test it?

File: Kubernetes/README.md (hunk 1 above)
Comment (bodo-sai) [reply Kubernetes/README.md:49]: Users can name it whatever they want to name the release

Comment (sahil1105) [reply Kubernetes/README.md:49]: Ok, let's add an example.

File: Kubernetes/helm/crds/mpi-operator.yaml (hunk 2 above)
Comment (bodo-sai) [reply Kubernetes/helm/crds/mpi-operator.yaml:39]: This is the CRD for MPIJob

File: Kubernetes/README.md (hunk 1 above)
Comment (bodo-sai) [reply Kubernetes/README.md]: added it

**Comment by sahil1105** [comment]
> > Looks good overall. Thanks @bodo-sai.
> > Have we asked someone to test it?
> 
> I have tested it in a new K8s cluster.

Sounds good. Let's also get someone else to test it. Maybe Karthik or Ali?

File: Kubernetes/README.md [hunk 3]
```diff
@@ -4,24 +4,77 @@ Bodo workloads can be deployed with Kubernetes using the [Kubeflow MPI-Operator]
...
+Check if Helm is installed in your system using `helm version`. If not, check this [guide](https://helm.sh/docs/intro/install/) for setting up Helm in your system.
+
+To run the sample bodo example, go to the repository and run the following command
+```
+helm install <release-name> <helm-directory-path>
+```
+For example, you can run the following for chicago crimes example
+
+```
+helm install chicago-crime Kubernetes/helm
```
Comment (sahil1105) [inline Kubernetes/README.md:53]:
```suggestion
helm install bodo-job-chicago-crime Kubernetes/helm
```

File: Kubernetes/helm/crds/mpi-operator.yaml (hunk 2 above)
Comment (sahil1105) [reply Kubernetes/helm/crds/mpi-operator.yaml:39]: Do we need all of them? My main concern is whether we should support all of them or just one?

File: Kubernetes/helm/values.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# Default values for bodo-k8s-chart.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+name: chicago-crime-example
+image: bodoaidocker/kube-mpioperator-minimal:latest
+
+slotsPerWorker: 2
```
Comment (sahil1105) [inline Kubernetes/helm/values.yaml:8]: Should this be the same as `cpu` on line 19?

**Review by sahil1105 — APPROVED** [review]
Thanks @bodo-sai! Looks good to me in general. See comments/questions/suggestions.

File: Kubernetes/helm/crds/mpi-operator.yaml (hunk 2 above)
Comment (bodo-sai) [reply Kubernetes/helm/crds/mpi-operator.yaml:39]: I think we need all of them.

File: Kubernetes/helm/values.yaml (hunk 4 above)
Comment (bodo-sai) [reply Kubernetes/helm/values.yaml:8]: I still don't understand the use of SlotsPerworker. I tried to understand  but still couldn't get any information regarding that.

**Review by alifarhid — APPROVED** [review]
Awesome! Thanks @bodo-sai

File: Kubernetes/helm/values.yaml (hunk 4 above)
Comment (sahil1105) [reply Kubernetes/helm/values.yaml:8]: What do the docs say?

**Pull request merged**